lib/webpacker/manifest.rb

```diff
@@ -13,32 +13,95 @@ def file_path
       Webpacker::Configuration.manifest_path
     end
 
+    # Throws an error if the file is not found. If Configuration.compile? then compilation is invoked
+    # the file is missing.
+    # React on Rails users will need to set Configuration.compile? to false as compilation is configured
+    # in the package.json for React on Rails.
```

Comment:
Specific configuration instructions for React on Rails should happen in React on Rails. We're not including config instructions for other integration packages here either.